### ISSUE TITLE
`pscale auth login` fails on a new machine 

### ISSUE THREAD

**Issue by threepointone** [planetscale/cli#596]
I'm setting up on my laptop. I installed `pscale` via `brew`. Then, running `pscale auth login` does the oauth dance with the browser fine, but then fails with the error: 

```
~ pscale auth login

Confirmation Code:<code>

If something goes wrong, copy and paste this URL into your browser: https://auth.planetscale.com/oauth/device?user_code=<coid>

Successfully logged in.
Error: open /Users/threepointone/.config/planetscale/pscale.yml: no such file or directory
```

This happened multiple times. I fixed it by creating the `/Users/threepointone/.config/planetscale` folder, so I suspect the bug is that the cli is trying to create the `pscale.yml` file without first checking whether the directory exists.

**Comment by pogilvie** [comment]
I'm hitting the same issue.   Thank you for the workaround

**Comment by GyoHeonLee** [comment]
I have same issue too.

macOS Ventura-13.0.1

**Comment by jerriclynsjohn** [comment]
I don't think this issue must be closed, I'm still facing this problem. Used the workaround and created a folder under `.config`, and then it worked. @fooforge . I'm on WSL2
